src/atoms/typography/heading.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { HeadingPriority } from "../constants/heading-priority";
+
+// -----------------------------------------------------------------------------------------
+// #region Interfaces
+// -----------------------------------------------------------------------------------------
+
+export interface HeadingProps {
+    children?: any;
+    cssClassName?: string;
+    id?: string;
+    priority?: HeadingPriority;
+}
+
+// #endregion Interfaces
+
+// -----------------------------------------------------------------------------------------
+// #region Component
+// -----------------------------------------------------------------------------------------
+
+const Heading: React.FC<HeadingProps> = (props: HeadingProps) => {
+    let cssClassNames: Array<any> = [];
+
+    if (props.cssClassName) {
+        cssClassNames.push(props.cssClassName);
+    }
+
+    let componentProps = {
+        className: cssClassNames.join(" "),
+    };
+
+    return React.createElement(
+        `h${props.priority || HeadingPriority.Two}`,
```

Comment:
Nitpick - prefer `??` vs `||` for null coalescing. (Assuming this library is on the version of CRA/Typescript that supports the operator)

Reply:
Agreed. Believe this comes from when the project was an older version of TS. Pre-port it was 3.7 and this project is up on 3.8